exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -17,11 +17,7 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.ServerSocket;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
It's not recommended to use wild card imports.
IntelliJ has a setting to disable this: https://stackoverflow.com/a/3348855